source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -3068,7 +3068,7 @@ void SwiftLanguageRuntime::FindFunctionPointersInCall(
                   ExecutionContext exe_ctx;
                   frame.CalculateExecutionContext(exe_ctx);
                   error = argument_values.GetValueAtIndex(0)->GetValueAsData(
-                      &exe_ctx, data, 0, NULL);
+                      &exe_ctx, data, NULL);
```

Comment:
This one is unrelated to the GetSymbolVendor removal ... the API to GetValueAsData was changed.